packages/experiments/src/components/UnifiedPicker/UnifiedPeoplePicker/UnifiedPeoplePicker.tsx

```diff
@@ -18,12 +18,12 @@ export const UnifiedPeoplePicker = (props: IUnifiedPeoplePickerProps): JSX.Eleme
     return <SelectedPeopleList {...selectedPeopleListProps} ref={null} />;
   };
   return (
-    <>
+    <div>
```

Comment:
I don't understand the reason for adding a wrapping div here? (or in some of the other examples, but especially this one because it doesn't even have a role)

Reply:
The reason to add div in other places was to just give them a role. This one definitely needs to be removed